listener/ronin.go

```diff
@@ -294,6 +295,46 @@ func (l *RoninListener) WithdrewCallback(fromChainId *big.Int, tx bridgeCore.Tra
 	return l.bridgeStore.GetTaskStore().Save(ackWithdrewTask)
 }
 
+func (l *RoninListener) ExternalCallFailedCallback(fromChainId *big.Int, tx bridgeCore.Transaction, data []byte) error {
+	if !l.config.EnableSyncReward {
+		return nil
+	}
+
+	failedEvent := new(bridgeTracking.BridgeTrackingExternalCallFailed)
+	bridgeTrackingAbi, err := bridgeTracking.BridgeTrackingMetaData.GetAbi()
+	if err != nil {
+		return err
+	}
+
+	if err = l.utilsWrapper.UnpackLog(*bridgeTrackingAbi, failedEvent, "ExternalCallFailed", data); err != nil {
+		return err
+	}
+
+	// We only process ExternalCallFailed event with to = bridge reward and
+	// msgSig = IBridgeReward.execSyncReward.selector
+	// execSyncReward(address[],uint256[],uint256,uint256,uint256)
+	var execSyncRewardSelector = [4]byte(common.Hex2Bytes("6bcb6fd6"))
+	if failedEvent.To != common.HexToAddress(l.config.Contracts[task.BRIDGE_REWARD_CONTRACT]) {
+		return nil
```

Comment:
add a log Infor here for tracking skipping syncRewardTask with `failedEvent.To` which is not `BRIDGE_REWARD_CONTRACT`

Reply:
We don't handle that event, why do we need to log it?

Reply:
Hm, my idea is to try to log some cases like this like storing logs for investigating in the future if some logic doesn't happen correctly in Prod. But if u think, this log is useless, then skip my comment.

Reply:
Actually, the correct way is that these 2 fields are indexed in contract and we query the event with the correct value of these 2 fields. However, the current library does not support that case so we just query all events then filter ourselves.